pgbouncer/datadog_checks/pgbouncer/pgbouncer.py

```diff
@@ -167,15 +167,20 @@ def check(self, instance):
     def _set_metadata(self):
         if self.is_metadata_collection_enabled():
             pgbouncer_version = self.get_version()
-            self.set_metadata('version', pgbouncer_version)
+            if pgbouncer_version:
+                self.set_metadata('version', pgbouncer_version)
+            else:
+                self.log.debug("Couldn't detect version")
```

Comment:
Might be more useful to log the raw result of `cursor.execute('SHOW VERSION;')` in `get_version()` method when version can't be detected ?

Reply:
Yep done.